cms/djangoapps/contentstore/views/course.py

```diff
@@ -546,13 +547,48 @@ def format_library_for_view(library):
     split_archived = settings.FEATURES.get(u'ENABLE_SEPARATE_ARCHIVED_COURSES', False)
     active_courses, archived_courses = _process_courses_list(courses_iter, in_process_course_actions, split_archived)
     in_process_course_actions = [format_in_process_course_view(uca) for uca in in_process_course_actions]
+    libraries = [format_library_for_view(lib) for lib in libraries]
+
+    courses_page = int(request.GET.get('courses_page', 1))
+    archived_courses_page = int(request.GET.get('archived_courses_page', 1))
+    libraries_page = int(request.GET.get('libraries_page', 1))
+
+    items_visible = getattr(settings, 'STUDIO_ITEMS_VISIBLE_DASHBOARD', 10)
```

Comment:
Hi @huynguyen-groove ,
Please help me to rename the variables:

- courses_page => page_number_courses
- archived_courses_page => page_number_archived_courses
- libraries_page => page_number_libraries
- STUDIO_ITEMS_VISIBLE_DASHBOARD  => ITEMS_ON_STUDIO_DASHBOARD